### DISCUSSION TITLE
Improve color contrast for accessibility

### DISCUSSION THREAD

**Discussion by slhck** [community/community#38651] [Accessibility] [answered]
Right now, the color contrast for Code Search is rather low, especially compared to the main GitHub page.

<img width="1992" alt="image" src="https://user-images.githubusercontent.com/582444/200768624-c1b98433-abbe-4502-95b3-aa5d7f206f7a.png">

Could these colors be harmonized? Especially code is hard to read, using light gray on dark gray:

<img width="157" alt="image" src="https://user-images.githubusercontent.com/582444/200768734-bc92a89e-aa7f-4f4d-a106-c55fefee4747.png">

**Comment by cehfisher** [comment, marked as answer]
Hi @slhck - thank you for providing this feedback! After reviewing your issue report against the latest UI, this issue appears to be resolved now. Could you please confirm and close this issue? Thank you!

**Reply by slhck** [reply]
Thanks for the reply. If by closing this issue you mean selecting an answer, sure! This was affecting the old version of code search.
